CMakeLists.txt

```
@@ -52,6 +52,12 @@ endif()
 
 set(CMAKE_CXX_STANDARD 17)
 
+
+if (MSVC)
+	# NOTE: STD ON MSVC DEBUG BUILDS IS ABSOLUTELY SLOW WHEN THE ITERATOR DEBUGGGING SUPPORT IS ENABLED. 
```

Comments should be in lower case, and as simple as possible
```suggestion
	# Disable iterator debug, improves performance of debug builds
```